src/main/resources/docker/Dockerfile

```diff
@@ -11,6 +11,8 @@ RUN groupadd -g 1000 -r sld \
  && chown -R sld:0 /home/sld /var/lm /data \
  && chmod -R 775 /home/sld \
  && chmod -R 775 /var/lm
+RUN apt update -y \
+ && apt upgrade -y
```

Comment:
Running apt commands without cleaning up afterwards can inflate the image size significantly. You should clean up by running `rm -rf /var/lib/apt/lists/*` as the last step in that run command to make sure the image doesn't contain the apt cache.

See: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run for more reference and sample docker commands to do this.

Reply:
@hhellyer : Done the changes as per your review comment. Kindly review the same and let us know if any further code modification is needed.